models/stdp_synapse_facetshw_hom_impl.h

```diff
@@ -123,7 +123,7 @@ void
 STDPFACETSHWHomCommonProperties< targetidentifierT >::calc_readout_cycle_duration_()
 {
   readout_cycle_duration_ = int( ( no_synapses_ - 1.0 ) / synapses_per_driver_ + 1.0 ) * driver_readout_time_;
-  // std::cout << "stdp_connection_facetshw_hom::debug: readout cycle duration
+  // std::cout << "stdp_synapse_facetshw_hom::debug: readout cycle duration
```

Comment:
I think we can just remove this debugging comment. People can just add it again if they need it.